From 201c4684062bae78bdf4875402ad88dbdf457681 Mon Sep 17 00:00:00 2001
From: Fabien Dessenne <fabien.dessenne@st.com>
Date: Mon, 25 May 2015 14:03:03 +0200
Subject: [PATCH 3/4] drm: add DRM_MODE_FB_BFF flag definition

Alignment to kernel drm API update : Add DRM_MODE_FB_BFF flag definition

Change-Id: Iaaa7b141ad3268901b65d483cfffc6c9a9fa7508
Signed-off-by: Fabien Dessenne <fabien.dessenne@st.com>
Reviewed-on: https://gerrit.st.com/30337
Reviewed-by: Vincent ABRIOU <vincent.abriou@st.com>
Reviewed-by: Hugues FRUCHET <hugues.fruchet@st.com>
Reviewed-on: https://gerrit.st.com/46619
Reviewed-by: Christophe PRIOUZEAU <christophe.priouzeau@st.com>
---
 include/drm/drm_mode.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/drm/drm_mode.h b/include/drm/drm_mode.h
index 115f36e..8c39c98 100644
--- a/include/drm/drm_mode.h
+++ b/include/drm/drm_mode.h
@@ -324,6 +324,7 @@ struct drm_mode_fb_cmd {
 
 #define DRM_MODE_FB_INTERLACED	(1<<0) /* for interlaced framebuffers */
 #define DRM_MODE_FB_MODIFIERS	(1<<1) /* enables ->modifer[] */
+#define DRM_MODE_FB_BFF		(1<<2) /* if interlaced, bottom field first */
 
 struct drm_mode_fb_cmd2 {
 	__u32 fb_id;
-- 
1.9.1

